test/acceptance/integration/terrain.py

```diff
@@ -60,7 +60,7 @@ def before_all_scenarios():
     user_roles_general = [(world.ks['user_all'], x['name']) for x in
                           world.ks['environment_general']['domains'][0]['users'][0]['projects'][0]['roles']]
     initialize_ac(user_roles_general,
-                  world.ac['ip'],
+                  world.ac['ip'], world.ac['port'],
```

Comment:
Do you think having two attributes in the same line is compliant with Python's style guidelines? (I'm not an expert in the matter, but I'd introduce a break there).